kafka_consumer_freshness_tracker/src/main/java/com/tesla/data/consumer/freshness/ConsumerFreshness.java

```diff
@@ -129,6 +154,51 @@ private void setup(Map<String, Object> conf) {
     this.executor = MoreExecutors.listeningDecorator(Executors.newFixedThreadPool(workerThreadCount));
   }
 
+  /**
+   * Validate an individual cluster's configuration. A cluster's configuration is valid if:
+   * a) Looking it up by name in Burrow returns a successful response
+   * b) Bootstrap servers match those in burrow
+   *  i. in strict mode: The bootstrap server list in the config and the list returned from Burrow contain the same set
+   *  of servers
+   *  ii. in normal mode: All entries in the config's bootstrap server list are advertised by Burrow (Burrow could
+   * have additional entries which don't appear in the config).
+   *
+   * @param clusterConf the cluster configuration
+   * @return an optional string which will either contain the validation failure message if validation fails or be empty
+   * if validation succeeds
+   */
+  private Optional<String> validateClusterConf(Map<String, Object> clusterConf) {
+    final String clusterName = (String) clusterConf.get("name");
+    final Set<String> bootstrapServersFromBurrow;
+    try {
+      bootstrapServersFromBurrow = new HashSet<>(this.burrow.getClusterBootstrapServers(clusterName));
+    } catch (IOException e) {
+      this.metrics.burrowClusterDetailReadFailed.labels(clusterName).inc();
+      return Optional.of("failed to read cluster detail from Burrow: " + e.getMessage());
+    }
+
+    final Map<String, String> clusterConfKafkaSection = (Map<String, String>) clusterConf.get("kafka");
+    final Set<String> bootstrapServersFromConfig = Arrays
+            .stream(clusterConfKafkaSection.get("bootstrap.servers").split(","))
+            .map(String::trim)
```

Comment:
feels like trimming could be slightly misleading - lets just give the people back what they gave us

Reply:
well if we don't trim then we need to mandate that they either use comma or comma-space to separate the servers in their list, with the trim we can be flexible in that regard.

Reply:
We already expect a single string as the value of the bootstrap servers. That same string is passed to kafka. We should just present that here - exactly what they configured.

Reply:
done